seuac-credentials-plugin/README.md

```diff
@@ -43,6 +43,7 @@ The plugin defines the following tasks:
 Task name | Depends on | Type | Description
 --- | --- | --- | ---
 `setCredentials`| - | - | Sets a credential. Invoke with `--key [Key of the credentials]` parameter.
+`clearCredentials`|-|-|Clears a credential. Invoke with `--key [Key of the credentials]` parameter. Invoke without `--key` parameter to clear all stored credentials.
```

Comment:
I don't think the behaviour without the --key argument is a good one. One typo and all your credentials are gone, without confirmation.